readit/readit/settings.py

```diff
@@ -78,8 +80,12 @@ def print_smth():
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR / 'db.sqlite3',
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'myproject',
```

Comment:
What is the purpose of the 'NAME'? Isn't it the name of the database in the postgres server or it's something else?

Reply:
Yes, it's the name of the database. I didn't know what to put in there and that's why I left it like this.

Reply:
The database name in our server is `readit`, I suppose it's critical to be filled-in in the settings. 

Please provide the env file in the Google Drive directory, so that everyone can download it locally in his working dir.

Reply:
Last changes: 6e5d7c4
- I added the 'NAME' to the .env file and corrected it in the settings.py file
- I sent the information from the file to the others from the team and added it to the Google Drive directory

Reply:
Great! The next thing to do is to unite the both frameworks into one single project dir and make them work togheter. 
@Stake-And-Rope/frontend-team @Stake-And-Rope/backend-team assemble a small team of max 3-4 people who will work on that task, set the Issue in the github project and try to implement both the frameworks in one project.